### ISSUE TITLE
On the beta download tab simplify the signature verification

### ISSUE THREAD

**Issue by NottyCode** [OpenLiberty/openliberty.io#2916]
The signature verification for betas says:

    Starting with version 22.0.0.1, signature files are produced for every package of an Open Liberty release. You can use these 
    signature files and the corresponding public key to verify the authenticity and integrity of an Open Liberty release package. 
    For more information, see [Verifying Open Liberty release packages.](https://openliberty.io/docs/22.0.0.1/verifying-package-signatures.html) 

    [Obtain the public key from this link.](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/sign/public_keys/WebSphereLiberty_06-02-2021.pem) Save the file from your browser as a .pem file.

however since there is only 1 beta available to download it doesn't make sense to talk about this in terms of a change. Lets identify a more compact way to express this.

**Comment by kinueng** [comment]
@mbroz2 @dmuelle FYI

**Comment by dmuelle** [comment]
proposed revision:

> Signature files are produced for each Open Liberty release package. You can use these 
signature files and the corresponding public key to verify the authenticity and integrity of an Open Liberty beta release package. 
For more information, see [Verifying Open Liberty release packages.](https://openliberty.io/docs/22.0.0.1/verifying-package-signatures.html) 

> [Obtain the public key from this link.](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/sign/public_keys/WebSphereLiberty_06-02-2021.pem) Save the file from your browser as a .pem file.

**Comment by mbroz2** [comment]
Could make it as short as:

> To verify the authenticity and integrity of an Open Liberty release package, use the provided signature file (SIG) and [public key](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/sign/public_keys/WebSphereLiberty_06-02-2021.pem). For more information, see [Verifying Open Liberty release packages.](https://openliberty.io/docs/latest/verifying-package-signatures.html)

**Comment by dmuelle** [comment]
Should we retain the instruction to save it as a .pem file? That's covered in the linked topic but someone might click directly through to DL the key without reading it. Also could specify "beta" here since this is on a tab just for the beta DL

> To verify the authenticity and integrity of an Open Liberty beta release package, use the provided signature file (SIG) and [public key](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/sign/public_keys/WebSphereLiberty_06-02-2021.pem). Save the public key file from your browser as a `.pem` file. For more information, see [Verifying Open Liberty release packages.](https://openliberty.io/docs/latest/verifying-package-signatures.html)

**Comment by mbroz2** [comment]
I don't think we need to specify 'beta' for the reason that they're already on the beta tab, and because the above applies regardless of whether it's beta or release.

Regarding the .pem extension, I'm largely impartial.  I think most users looking to do this, would either know to save it as `.pem` or figure it out from the URL which shows that it's a `.pem` file.  For those that don't, I'd expect they're going to need the documentation for how to verify which would explicitly tell them to save it as `.pem`

In other words, the reword provides the location of the necessary bits and the docs provide the instruction for what to do with them for those that don't know know.  I'm not against keeping the part about the extension, but I'm also not sure we need it when it's already explained in the doc.

**Comment by dmuelle** [comment]
sounds good. Should we also update the main release tab to be more concise?

I also just noticed in the doc url above, the release version is hard coded. Michal's revision fixes this problem but it needs to be fixed on the main release tab as well.

https://openliberty.io/docs/22.0.0.1/verifying-package-signatures.html
--->
https://openliberty.io/docs/latest/verifying-package-signatures.html

**Comment by mbroz2** [comment]
regarding the `latest` in the URL, yes, I was going to make sure that was going to get fixed on the release tab as well. 

However, on the release tab, I think we need to keep the 'Starting with 22.0.0.1' part.  That way it's clear that for the older versions it's not a mistake that they're not available. So maybe something like:
> To verify the authenticity and integrity of an Open Liberty 22.0.0.1 release package or newer, use the provided signature file (SIG) and [public key](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/sign/public_keys/WebSphereLiberty_06-02-2021.pem). For more information, see [Verifying Open Liberty release packages.](https://openliberty.io/docs/latest/verifying-package-signatures.html) 

Though I'm guessing you might have different wording for the above :)

**Comment by dmuelle** [comment]
maybe:

> In version 22.0.0.1 and later, to verify the authenticity and integrity of an Open Liberty release package, use the provided signature file (SIG) and [public key](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/sign/public_keys/WebSphereLiberty_06-02-2021.pem). For more information, see [Verifying Open Liberty release packages.](https://openliberty.io/docs/latest/verifying-package-signatures.html)

**Comment by mbroz2** [comment]
either or is good with me.
Is 'later' preferred over 'newer'?  I've previously noticed we use 'later' but my brain always expects 'newer'..... maybe it has to do with me associating 'later' with the negative aspect of something being late, and 'newer' with fun shinny new things

**Comment by dmuelle** [comment]
We generally use later/earlier in the docs as opposed to newer/older but it's not a hard rule.

**Comment by mbroz2** [comment]
![image](https://user-images.githubusercontent.com/3076261/215181595-54f9dd40-a31e-4d4f-b569-f516e39b84c8.png)
![image](https://user-images.githubusercontent.com/3076261/215181663-7a22f810-de82-4ef0-ae95-114c6e961b54.png)
